package.json

```diff
@@ -50,6 +50,8 @@
   "homepage": "https://github.com/SU-SWS/decanter-react#readme",
   "dependencies": {
     "@heroicons/react": "^1.0.4",
+    "@mui/core": "^5.0.0-alpha.49",
+    "@mui/material": "^5.0.2",
```

Comment:
Please remove this package.

Reply:
I'm using this for styling the focused item in the dropdown, '@mui/material/styles';. They have some custom magic going on that doesn't work with focus: tailwind styles, so I can apply hover styles but for focus I have to target their specific class and this seems to be mui's native way of doing things